### ISSUE TITLE
Missing country code on onlyCountry list breaks component

### ISSUE THREAD

**Issue by thiagosbrito** [tanansatpal/ngx-mat-intl-tel-input#73]
Hello, 

I'm facing an issue when I have a phone number which its country code is not listed on the `onlyCountries` array, it breaks the page and throws the following error:

```
ERROR TypeError: Cannot read property 'flagClass' of undefined
    at Object.eval [as updateDirectives] (NgxMatIntlTelInputComponent.html:3)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:45259)
    at checkAndUpdateView (core.js:44271)
    at callViewAction (core.js:44637)
    at execComponentViewsAction (core.js:44565)
    at checkAndUpdateView (core.js:44278)
    at callViewAction (core.js:44637)
    at execComponentViewsAction (core.js:44565)
    at checkAndUpdateView (core.js:44278)
    at callViewAction (core.js:44637)
```
I understand that if the country code is not present on `onlyCountries`, the flag is not found, although IMHO it should return a "non-country" image, like a placeholder instead of throwing an error and prevent component to keep working as expected

**Comment by tanansatpal** [comment]
Hi @thiagosbrito , this seems an edge case, but I think it would be easy to handle. So, I was thinking maybe we show the UN flag no dialCode. 
But it would have one drawback that, it would come in the list.

**Comment by thiagosbrito** [comment]
Hi @tanansatpal, thanks for the reply. I understand your point. I think just a class if the country code is missing on the onlyCountries would be enough

**Comment by tanansatpal** [comment]
Hi @thiagosbrito , this is fixed in the latest release 3.3.0
